app/src/modules/settings/routes/data-model/new-collection.vue

```diff
@@ -212,6 +221,10 @@ export default defineComponent({
 			singleton,
 		};
 
+		function setOptionsForSingleton() {
+			systemFields.sort.inputDisabled = singleton.value;
```

Comment:
We may also need to turn `systemFields.sort.enabled` to false if they enabled it first before they check the singleton checkbox. Not sure how likely will a user do this, but should still be a possibility 🤔

Reply:
Thank you @azrikahar 
Definitely a use case that could happen and I didn't remember ;)